src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java

```diff
@@ -21,14 +21,17 @@
   @FindBy(css = ".sort span")
   private WebElement labelInSortEntryPointMobile;
 
-  @FindBy(css = ".discussion-sort")
+  @FindBy(css = ".pop-over-container")
   private WebElement sortOptionsMobile;
 
-  @FindBy(xpath = "//li[text()='Latest']")
-  private WebElement latestLinkOnListMobile;
+  @FindBy(css = ".sort-options li:nth-child(1) label")
```

Comment:
We might add new ways of sorting posts in the future or change their order. Would it be possible to find these two by ID instead of using nth-child? ID's would be *sort-button-main.sort-by-trending* and *sort-button-main.sort-by-lates*?